src/screens/EventDetailsScreen/component.js

```diff
@@ -122,7 +108,10 @@ class EventDetailsScreen extends PureComponent<Props> {
             source={{ uri: getAssetUrl(event.fields.individualEventPicture) }}
           />
           <EventOverview event={event} />
```

Comment:
I think you could remove the ContentPadding from the EventOverview, and move EventOverview inside the ContentPadding defined here.

Reply:
Yess!!